### PR TITLE
Optimazing time tests

### PR DESCRIPTION
**Pull request by luispresuelVenafi**
Closes VC-24086

### CODE REVIEW

File: plugin/util/constants.go [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+const (
+	CredentialsRootPath                          = `venafi/`
+	tokenMode                                    = `TPP Token (access_token, refresh_token)` // #nosec G101
+	tppMode                                      = `TPP Credentials (tpp_user, tpp_password)`
+	cloudMode                                    = `Cloud API Key (apikey)`
+	StoreByCNString                              = "cn"
+	StoreByHASHstring                            = "hash"
+	StoreBySerialString                          = "serial"
+	ErrorTextValueMustBeLess                     = `"ttl" value must be less than "max_ttl" value`
+	ErrorTextStoreByAndStoreByCNOrSerialConflict = `Can't specify both story_by and store_by_cn or store_by_serial options '`
```
[the remark this answers is not in the document]
Comment (luispresuelVenafi) [reply plugin/util/constants.go:14]: Before they were scattered a long all of the files, the idea was to order this text in a place. Later we can focus on this refactor for errors in other PR. WDYT?

File: plugin/pki/test_env.go [hunk 2]
```diff
@@ -2821,28 +2821,44 @@ func (e *testEnv) NegativeIssueCertificateAndSaveSerial(t *testing.T, data testD
...
-	return &testEnv{
-		Backend:          b,
-		Context:          ctx,
-		Storage:          config.StorageView,
-		TestRandString:   randSeq(9),
-		RoleName:         randSeq(9) + "-role",
-		VenafiSecretName: randSeq(9) + "-venafi",
-	}, nil
-}
+//type NEWPKI interface {
```
[the remark this answers is not in the document]
Comment (luispresuelVenafi) [reply plugin/pki/test_env.go:2824]: Good catch, this is a left over, will remove by next commit

[the remark this answers is not in the document]
Comment (luispresuelVenafi) [reply plugin/pki/test_env.go]: Removed by latest commit

**Review by rvelaVenafi — APPROVED** [review]
Ok

**Review by marcos-albornoz — APPROVED** [review]

**Pull request merged**